app/helpers/accounts_helper.rb

```diff
@@ -38,6 +40,8 @@ def edit_account_form_url(account)
     case account.accountable_type
     when "Property"
       property_path(account)
+    when "Vehicle"
+      vehicle_path(account)
```

Comment:
We will eventually not need this.  This is an interim helper so that while we're developing each view, we don't have to do one big reorg of files and views.